src/component/home/home.component.jsx

```diff
@@ -1,112 +1,115 @@
-    const {value} = e.target
-    setName(value)
-    console.log(value)
-  }
-//Snapchot for firebase with for each
-  useEffect(
-    () => {
-      db.collection("hahahe").onSnapshot(snap => {
-        let list = []
+    const { value } = e.target;
+    setName(value);
+    console.log(value);
+  };
+  //Snapchot for firebase with for each
+  useEffect(() => {
+    db.collection("hahahe").onSnapshot((snap) => {
+      let list = [];
 
-        snap.forEach((doc) => {
-          list.push({
-            id: doc.id,
-            ...doc.data()
-          })
-        })
-        setNameList(list)
-      })
-    },
-    []
-  )
+      snap.forEach((doc) => {
+        list.push({
+          id: doc.id,
+          ...doc.data(),
+        });
+      });
+      setNameList(list);
+    });
+  }, []);
 
-  const handlePress = e => {
+  const handlePress = (e) => {
     if (e.charCode === 13) {
       // Vreau sa il trimit la baza de date cand apas enter
       db.collection("hahahe").add({
         name,
-        anaaremere: Math.random() - 0.5 > 0
-      })
+        anaaremere: Math.random() - 0.5 > 0,
+      });
       // Vreau sa curat formularul
-      setName("")
-      e.target.focus()
+      setName("");
+      e.target.focus();
     }
-    console.log(e, Object.keys(e))
-  }
+    console.log(e, Object.keys(e));
+  };
 
-  const dispatch = useDispatch()
+  const dispatch = useDispatch();
 
-  const isLoggedIn = useSelector(({ user }) => user.isLoggedIn)
+  const isLoggedIn = useSelector(({ user }) => user.isLoggedIn);
 
   const smartRandom = () => {
     db.collection("haha").add({
-      name: Math.random()-0.5>0 ? "Hihi" : "haHa",
-      areanamere: !isLoggedIn
-    })
-  }
+      name: Math.random() - 0.5 > 0 ? "Hihi" : "haHa",
+      areanamere: !isLoggedIn,
+    });
+  };
 
   //Dispatch the action to the store(to the reducer)
   const logheazaMa = () => {
     //Pentru test vreau sa vad cum se schimba store vis a vis de isLoggedIn
-    dispatch(logheazaMaAction())
-  }
+    dispatch(logheazaMaAction());
+  };
 
   const scoateMa = () => {
     //Pentru test vreau sa vad cum se schimba store vis a vis de isLoggedIn
-    dispatch(scoateMaAction())
-  }
-
-  console.log(isLoggedIn)
-
-  console.log(nameList)
+    dispatch(scoateMaAction());
+  };
 
-  return(
+  return (
     <HOS.HomeContainer>
       <HOS.Poster>
         <HOS.PosterContent>
-          <HOS.PosterText>
-          The best place to buy and sell!
-          </HOS.PosterText>
-          {!isLoggedIn && <HOS.PosterButton onClick={logheazaMa}>Register</HOS.PosterButton>}
-          {isLoggedIn && <HOS.PosterButton onClick={scoateMa}>LogOUT</HOS.PosterButton>}
-          <HOS.PosterButton onClick={smartRandom}>Add Random Data</HOS.PosterButton>
+          <HOS.PosterText>The best place to buy and sell!</HOS.PosterText>
+          {!isLoggedIn && (
+            <HOS.PosterButton onClick={logheazaMa}>Register</HOS.PosterButton>
+          )}
+          {isLoggedIn && (
```

Comment:
No need to check isLoggedIn twice, we can use a ternary here. 
{!isLoggedIn 
? <HOS.PosterButton onClick={logheazaMa}>Register</HOS.PosterButton> 
: <HOS.PosterButton onClick={scoateMa}>LogOUT</HOS.PosterButton>}
}